core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.scss

```
@@ -52,8 +52,7 @@
     }
 
     .dot-pages-panel__expanded .p-toggleable-content .p-panel-content {
-        overflow: unset;
-        height: unset;
+        overflow: scroll;
```

The `overflow: scroll;` [displays scroll bars whether or not any content is overflowing or clipped](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#values). That means we will display a scroll bar even if there is no `overflowing`. Is that ok?

We can use `overflow: auto;` to avoid that behavior.

```suggestion
        overflow: auto;
```

---

Alright! I'll change it